tests/cxx/using_import.cc

```diff
@@ -0,0 +1,29 @@
+//===--- using_import.cc - test input file for iwyu -----------------------===//
+//
+//                     The LLVM Compiler Infrastructure
+//
+// This file is distributed under the University of Illinois Open Source
+// License. See LICENSE.TXT for details.
+//
+//===----------------------------------------------------------------------===//
+
+#include "using_import-import.h"
+
+void use_symbol() {  
+  // IWYU: ns::symbol is defined in ...*using_import-declare.h", which isn't directly #included.
+  symbol();
+}
```

Comment:
> Does this test catch something that is not caught by include_with_using.cc? It is kinda similar to include_with_using-d3.h, include_with_using-d3b.h. But it's not exactly the same and I don't know how important is the full use in this test.

It might, but I'm not certain because include_with_using.cc is, in my personal opinion, trying to do too much. I actually think many of the iwyu tests are guilty of this. I prefer to start with very simple, specific tests, and progress to more complicated cases that look more like integration tests, but that might just be me. At any rate, I was trying to isolate the specific case I was fixing while making sure I didn't break basic functionality.

> And please don't use 'import' in test name because there are also #import and @import. Test name in this case is misleading.

Ah, right.  I hadn't considered that.  I will find a better name.